File: CC_API/CC_API/ViewModels/ProfileVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CC_API
{
    public class ProfileVM
    {
        public ProfileVM()
        {
            this.UploadLineVMs = new HashSet<UploadLineVM>();
        }

        public int ProfileID { get; set; }
        public int Customer_ID { get; set; }
        public string CustomerName { get; set; }
        public int SkillID { get; set; }
        public string SkillName { get; set; }
        public string Bio { get; set; }

        public virtual ICollection<UploadLineVM> UploadLineVMs { get; set; }
    }
}

File: Creatives_Connect/src/app/accountprofiles/profile-vm.ts
import { UploadLineVM } from "./upload-line-vm";


export class ProfileVM {
    

    constructor(){
        this.UploadLineVMs = [];
    }

    ProfileID : number;
    Customer_ID : number;
    SkillID : number;
    SkillName : string;
    Bio : string;
    ProfilePic: string;
    Username: string;
    Pricing: string;

    UploadLineVMs : UploadLineVM[];


}
